tests/unit/iac/output/test_iac_text_output.py

```diff
@@ -124,6 +125,14 @@ def assert_no_failures_displayed(result: Result):
     assert "Error scanning. Results may be incomplete." not in result.stdout
 
 
+def assert_documentation_url_displayed(result: Result):
+    assert re.search(
+        r"\((GG_IAC_\d{4})\).+https://docs.gitguardian.com/iac-scanning/policies/\1",
```

Comment:
Nit: your regex is incorrect because some characters aren't escape such as the dots in the URL